sklearn/tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+'''

```

Comment:
To be honest, my personal feeling is that I would have that as a comment rather than a docstring. Basically I don't consider this piece of information a documentation for the sklearn.tests module. In other words if you do this:

```python
import sklearn.tests
sklearn.tests?
```

you will get the docstring which doesn't say anything about the sklearn.tests module and I would argue is a bit confusing.
will show the docstring which